src/main/scala/scala/concurrent/java8/FutureConvertersImpl.scala

```diff
@@ -66,13 +66,20 @@ object FuturesConvertersImpl {
       cf
     }
 
-    override def toCompletableFuture(): CompletableFuture[T] =
-      throw new UnsupportedOperationException("this CompletionStage represents a read-only Scala Future")
+    /**
+     * @inheritdoc
+     *
+     * WARNING: completing the result of this method will not complete the underlying
```

Comment:
Ah, now I see it, so you are using 2 words for the same Future, use `underlying` or `wrapped` consistently please :)